dbms/src/test/java/org/polypheny/db/sql/view/ComplexViewTest.java

```diff
@@ -2126,7 +1938,7 @@ public void testQ21() throws SQLException {
 
     // SELECT NOT POSSIBLE
     // Caused by: java.sql.SQLException: General error
-    @Ignore
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1519992631